src/sections/shared/add-data-actions/AddDataActionsButton.tsx

```diff
@@ -0,0 +1,45 @@
+import { Route } from '../../Route.enum'
+import { useTranslation } from 'react-i18next'
+import { Dropdown } from 'react-bootstrap'
+import { DropdownButton } from '@iqss/dataverse-design-system'
+import { PlusLg } from 'react-bootstrap-icons'
+import styles from './AddDataActionsButton.module.scss'
+
+const routesToRender = [
+  {
+    id: 0,
+    name: 'New Dataverse',
+    page: `/spa${Route.DATASETS}`,
+    disabled: true
+  },
+  {
+    id: 1,
+    name: 'New Dataset',
+    page: `/spa${Route.CREATE_DATASET}`,
+    disabled: false
+  }
+]
+
+export default function AddDataActionsButton() {
+  const { t } = useTranslation('header')
+  const list = routesToRender.map((option) => (
+    <Dropdown.Item href={option.page} disabled={option.disabled} key={option.id}>
+      {option.name}
+    </Dropdown.Item>
+  ))
+  return (
+    <div className={'d-flex justify-content-end mb-3'}>
```

Comment:
I think we can move the container div to the parent component: `<div className='d-flex justify-content-end mb-3'>`. This will allow any parent component that wants to reuse the "Add Data" button to choose the best alignment for each case.